plugins/AITagger/ai_tagger.py

```diff
@@ -0,0 +1,345 @@
+import os
+import sys
+import json
+import subprocess
+import csv
+from typing import Any
+
+# plugins don't start in the right directory, let's switch to the local directory
+os.chdir(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
chdir not necessary if you do relative path in .yml

Reply:
See above comment